src/webapp/azext_webapp/_help.py

```diff
@@ -14,3 +14,28 @@
     type: command
     short-summary: Creates a remote connection using a tcp tunnel to your web app
 """
+
+helps['webapp scan'] = """
+    type: group
+    short-summary: Holds group of commands which cater to webapp scans
+"""
+
+helps['webapp scan start-scan'] = """
+    type: command
+    short-summary: Starts the scan on the specified webapp files
```

Comment:
Can you give more clarity here - i.e how is this command going to be useful? Say Scans a specific web app - & what does it return?

Reply:
It will basically scan the wwwroot directory and will return a json containing the scan ID, tracking and result url.
Will add this in the short-summary.

Reply:
Thanks - basically we want the  customers to use this command so any information we can give them to tell them that this command  is useful - is going to be good to add.